.goreleaser.yml

```diff
@@ -17,6 +17,21 @@ archives:
     windows: Windows
     386: i386
     amd64: x86_64
+brews:
+  name: meroxa
+  tap:
+    owner: meroxa
+    name: homebrew-taps
+  homepage: https://meroxa.io
+  description: The Meroxa CLI
+  folder: Formula
+  license: "Apache 2.0"
+  custom_block: |
+    head "https://github.com/meroxa/cli.git"
+  install: |
+    bin.install "meroxa"
+  test: |
+    system bin/"meroxa", "version"
```

Comment:
Perhaps verifying the version to match the released version instead of only testing the command works. We already let goreleaser fill in the version: https://goreleaser.com/environment/#using-the-mainversion & https://github.com/meroxa/cli/blob/774ec28b1049d1627f989db503e9a176ce7fb35e/main.go#L22. And we can compare it to the git tag (`{{ .Tag }}`) which is available as part of the brew section: https://goreleaser.com/customization/homebrew/

Reply:
@owenthereal made [this change](https://github.com/meroxa/cli/pull/87/commits/3f6a62a01bf72cba44a125829b58daf0d6edb1d4), what do you think?